tests/parametric/test_otel_tracer.py

```diff
@@ -4,21 +4,20 @@
 from utils.parametric.spec.trace import find_span_in_traces
 from utils.parametric.spec.trace import find_span
 from utils.parametric.spec.otel_trace import OtelSpan
-from utils import missing_feature, irrelevant, context, scenarios
+from utils import missing_feature, context, scenarios
 
-# this global mark applies to all tests in this file.
 #   DD_TRACE_OTEL_ENABLED=true is required in some tracers (.NET, Python?)
 #   CORECLR_ENABLE_PROFILING=1 is required in .NET to enable auto-instrumentation
-pytestmark = pytest.mark.parametrize(
-    "library_env", [{"DD_TRACE_OTEL_ENABLED": "true", "CORECLR_ENABLE_PROFILING": "1"}],
-)
 
 
 @scenarios.parametric
 class Test_Otel_Tracer:
-    @irrelevant(context.library == "cpp", reason="library does not implement OpenTelemetry")
+    @missing_feature(context.library == "cpp", reason="library does not implement OpenTelemetry")
     @missing_feature(context.library == "ruby", reason="Not implemented")
     @missing_feature(context.library == "php", reason="Not implemented")
```

Comment:
If you are going to mark as "missing_feature" all methods in this class, you should remove these decorators and mark as "missing_feature" in the manifest files (ie manifests/ruby.yml)
For example:
`  parametric/:
    test_otel_tracer.py:
      Test_Otel_Tracer: missing_feature`